apps/stable/cluster-autoscaler/values.yaml.gotmpl

```diff
@@ -0,0 +1,28 @@
+{{- if eq "eks" .Values.jxRequirements.cluster.provider }}
+autoDiscovery: 
+  clusterName: {{ .Values.jxRequirements.cluster.provider }}
+  tags:
+  - k8s.io/cluster-autoscaler/enabled
+  - k8s.io/cluster-autoscaler/{{ .Values.autoDiscovery.clusterName }}
+awsRegion: {{ .Values.jxRequirements.cluster.region }}
+awsAccessKeyID: ""AKIAYBCHSJM224CUEI6T"
```

Comment:
whoa whoa! please remove these immediately and change your access keys. This is a public repo. IMMEDIATELY!